tests/depends/Trie/Test_Trie.cpp

```diff
@@ -67,410 +67,410 @@ BOOST_AUTO_TEST_CASE(fat_trie)
-        bytes r(t.rlp());
-        LOG_GENERAL(INFO, RLP(r));
-    }
-    {
-        MemTrie t;
-        t.insert("bed", "d");
-        t.insert("be", "e");
-        LOG_GENERAL(INFO, hex << t.hash256());
-        bytes r(t.rlp());
-        LOG_GENERAL(INFO, RLP(r));
-    }
-    {
-        LOG_GENERAL(
-            INFO,
-            hex << stringMapHash256({{"dog", "puppy"}, {"doe", "reindeer"}}));
-        MemTrie t;
-        t.insert("dog", "puppy");
-        t.insert("doe", "reindeer");
-        LOG_GENERAL(INFO, hex << t.hash256());
-        bytes r(t.rlp());
-        LOG_GENERAL(INFO, RLP(r));
-        LOG_GENERAL(INFO, toHex(t.rlp()));
-    }
-    {
-        MemoryDB m;
-        EnforceRefs r(m, true);
-        GenericTrieDB<MemoryDB> d(&m);
-        d.init(); // initialise as empty tree.
-        MemTrie t;
-        StringMap s;
-
-        auto add = [&](char const* a, char const* b) {
-            d.insert(string(a), string(b));
-            t.insert(a, b);
-            s[a] = b;
-
-            LOG_GENERAL(INFO, "/n-------------------------------");
-            LOG_GENERAL(INFO, a << " -> " << b);
-            LOG_GENERAL(INFO, d);
-            LOG_GENERAL(INFO, m);
-            LOG_GENERAL(INFO, d.root());
-            LOG_GENERAL(INFO, stringMapHash256(s));
-
-            BOOST_REQUIRE(d.check(true));
-            BOOST_REQUIRE_EQUAL(t.hash256(), stringMapHash256(s));
-            BOOST_REQUIRE_EQUAL(d.root(), stringMapHash256(s));
-            for (auto const& i : s)
-            {
-                (void)i;
-                BOOST_REQUIRE_EQUAL(t.at(i.first), i.second);
-                BOOST_REQUIRE_EQUAL(d.at(i.first), i.second);
-            }
-        };
-
-        auto remove = [&](char const* a) {
-            s.erase(a);
-            t.remove(a);
-            d.remove(string(a));
-
-            LOG_GENERAL(INFO, endl << "-------------------------------");
-            LOG_GENERAL(INFO, "X " << a);
-            LOG_GENERAL(INFO, d);
-            LOG_GENERAL(INFO, m);
-            LOG_GENERAL(INFO, d.root());
-            //            LOG_GENERAL(INFO, hash256(s));
-
-            BOOST_REQUIRE(d.check(true));
-            BOOST_REQUIRE(t.at(a).empty());
-            BOOST_REQUIRE(d.at(string(a)).empty());
-            BOOST_REQUIRE_EQUAL(t.hash256(), stringMapHash256(s));
-            BOOST_REQUIRE_EQUAL(d.root(), stringMapHash256(s));
-            for (auto const& i : s)
-            {
-                (void)i;
-                BOOST_REQUIRE_EQUAL(t.at(i.first), i.second);
-                BOOST_REQUIRE_EQUAL(d.at(i.first), i.second);
-            }
-        };
-
-        add("dogglesworth", "cat");
-        add("doe", "reindeer");
-        remove("dogglesworth");
-        add("horse", "stallion");
-        add("do", "verb");
-        add("doge", "coin");
-        remove("horse");
-        remove("do");
-        remove("doge");
-        remove("doe");
-    }
-}
+// BOOST_AUTO_TEST_CASE(hex_encoded_securetrie_test)
```

Comment:
A cleaner and more descriptive way to disable a test case: https://www.boost.org/doc/libs/1_59_0/libs/test/doc/html/boost_test/tests_organization/enabling.html

This at least will make this pr smaller - 500+ line changes can be removed. Also, don't forget to state the reason for disable the test case in the code (not in the pr discussion)